### ISSUE TITLE
Commit rows disappear after closing connection ?

### ISSUE THREAD

**Issue by markddrake** [MariaDB/mariadb-connector-nodejs#24]
I am sure I must be missing something really obvious here..
My application has just inserted 4 rows into the table "hr2"."regions"

I execute the following code 
```
    results = await conn.query('select count(*) from hr2.regions');
    console.log(results);
    await conn.commit;
    console.log('Committed');
    results = await conn.query('select count(*) from hr2.regions');
    console.log(results);
    
    await conn.end();

    conn = await pool.getConnection();
    results = await conn.query('select count(*) from hr2.regions');
    console.log(results);

```

And I get the following results..

```
[ { 'count(*)': 4 },
  meta: [ { collation: [Collation],
      columnLength: 21,
      columnType: 8,
      scale: 0,
      type: 'LONGLONG',
      flags: 129,
      db: [Function: bound getStringProperty],
      schema: [Function: bound getStringProperty],
      table: [Function: bound getStringProperty],
      orgTable: [Function: bound getStringProperty],
      name: [Function: bound getStringProperty],
      orgName: [Function: bound getStringProperty] } ] ]
Committed
[ { 'count(*)': 4 },
  meta: [ { collation: [Collation],
      columnLength: 21,
      columnType: 8,
      scale: 0,
      type: 'LONGLONG',
      flags: 129,
      db: [Function: bound getStringProperty],
      schema: [Function: bound getStringProperty],
      table: [Function: bound getStringProperty],
      orgTable: [Function: bound getStringProperty],
      name: [Function: bound getStringProperty],
      orgName: [Function: bound getStringProperty] } ] ]
[ { 'count(*)': 0 },
  meta: [ { collation: [Collation],
      columnLength: 21,
      columnType: 8,
      scale: 0,
      type: 'LONGLONG',
      flags: 129,
      db: [Function: bound getStringProperty],
      schema: [Function: bound getStringProperty],
      table: [Function: bound getStringProperty],
      orgTable: [Function: bound getStringProperty],
      name: [Function: bound getStringProperty],
      orgName: [Function: bound getStringProperty] } ] ]
```
How can 4 committed rows go AWOL. There are no other users of the database..

**Comment by rusher** [comment]
hmm, I don't know what I can say, but there is something wrong somewhere.
Those rows have been removed by some other part of the application, or application maybe use 2 phase commit (XA START / COMMIT ...) ?

**Comment by rusher** [comment]
ah, or someone (@vaintroub) point me that code : 
```javascript
 await conn.commit;
```

must be 

```javascript
 await conn.commit();
```

Thes doesn't return any error, but commit will not be executed. 

From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await:
> Returns the fulfilled value of the promise, or the value itself if it's not a Promise.

**Comment by rusher** [comment]
This is corrected in latest release (2.0.1) with https://github.com/MariaDB/mariadb-connector-nodejs/commit/50c758d7447cd0771041a4763527ad34ae29fcef

**Comment by markddrake** [comment]
Thank you for finding and fixing this. My missing rows are now where they belong, safely in the databsase. Since I'm new to MariaDB and MySQL I assumed I must have done something really stupid.. BTW I did have commit() rather than commit in my real code...
